modules/op-about-workspace.adoc

```diff
@@ -23,7 +28,7 @@ You can specify Workspaces in the TaskRun or PipelineRun using:
 * A PersistentVolumeClaim from a provided VolumeClaimTemplate
 * An emptyDir that is discarded when the TaskRun completes
 
-Here is a code snippet of the `build-and-deploy` Pipeline which declares a `shared-workspace` Workspace for the Tasks, `build-image` and `apply-manifests`, defined in the Pipeline.
+The following example shows a code snippet of the `build-and-deploy` Pipeline which declares a `shared-workspace` Workspace for the Tasks, `build-image` and `apply-manifests`, defined in the Pipeline.
```

Comment:
Commas in this sentence are a bit awkward; didn't catch it before. Suggestion:
The following example shows a code snippet of the `build-and-deploy` Pipeline, which declares a `shared-workspace` Workspace for the `build-image` and `apply-manifests` Tasks as defined in the Pipeline.